multik-api/src/commonMain/kotlin/org/jetbrains/kotlinx/multik/ndarray/data/MemoryView.kt

```diff
@@ -767,7 +779,7 @@ public class MemoryViewComplexFloatArray(override val data: ComplexFloatArray) :
 
     override fun equals(other: Any?): Boolean = when {
```

Comment:
You changed the above "= when" to "return when". Would like to stick to one style, at least in one file. 
Which one doesn't matter. probably better "return when"